src/config/config-set.ts

```diff
@@ -56,10 +56,15 @@ import { TSError } from '../utils/ts-error'
 
 const logger = rootLogger.child({ namespace: 'config' })
 
+interface AstTransformerObj<T = Record<string, unknown>> {
+  module: AstTransformerDesc
```

Comment:
hmm can we choose another name for this property, keyword `module` is kind of already occupied by js so I would avoid naming property like this.

Reply:
Any suggestions?

Reply:
hmm maybe `transformModule` ?